### PR TITLE
Update api reference for router outlet directive

### PR DESCRIPTION
**Pull request by jbogarthyde**
Incorporate more specific information about multiple outlets and how to target them, with link to tutorial example.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

API doc for `RouterOutlet` directive incomplete. 
https://angular.io/api/router/RouterOutlet

Issue Number: N/A

## What is the new behavior?

Adds discussion of named secondary outlets and their  usage.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/router/src/directives/router_outlet.ts [hunk 1]
```diff
@@ -27,6 +27,19 @@ import {PRIMARY_OUTLET} from '../shared';
  * <router-outlet name='right'></router-outlet>
  * ```
  *
+ * Named outlets can be the targets of secondary routes.
+ * The route configuration for a secondary route has a third property, `outlet:
```
Comment (atscott) [inline packages/router/src/directives/router_outlet.ts:31]: I _think_ what you mean by third property is (1)`path`, (2)`component`, and (3)`outlet` but in reality, a route configuration can have many more properties: https://angular.io/api/router/Route

File: packages/router/src/directives/router_outlet.ts [hunk 2]
```diff
@@ -27,6 +27,19 @@ import {PRIMARY_OUTLET} from '../shared';
...
+ * <target_outlet_name>`. Secondary routes are independent of each other, but work in combination
+ * with other routes. Using named outlets and secondary routes, you can target multiple outlets with
+ * multiple routes in the same `RouterLink` directive.
```
Comment (atscott) [inline packages/router/src/directives/router_outlet.ts:34]:
```suggestion
 * with other routes. Using named outlets and secondary routes, you can target multiple outlets in
 * the same `RouterLink` directive.
```

This sentence was really confusing to me. How about this instead?

File: packages/router/src/directives/router_outlet.ts [hunk 3]
```diff
@@ -27,6 +27,19 @@ import {PRIMARY_OUTLET} from '../shared';
...
+ *
+ * The router keeps track of separate branches in a navigation tree for each named outlet, and
```
Comment (atscott) [inline packages/router/src/directives/router_outlet.ts:36]:
```suggestion
 * The router keeps track of separate branches in a navigation tree for each named outlet and
```

**Review by atscott — CHANGES_REQUESTED** [review]

**Comment by jbogarthyde** [comment]
Thanks @atscott - changes made.

**Review by atscott — APPROVED** [review]

**Pull request closed without merging**